utils/common/src/main/kotlin/Utils.kt

```diff
@@ -128,3 +128,35 @@ fun <R> temporaryProperties(vararg properties: Pair<String, String?>, block: ()
         }
     }
 }
+
+/**
+ * Return a comma-separated string of values or ranges that represent the [current object].
+ */
+fun List<Pair<Int, Int>>.prettyPrintValues(): String {
```

Comment:
Please call this "prettyPrintRanges".

Reply:
Also, if this is an extension function now, it should actually go to `Extensions.kt`. Alternatively, make `collapseValues` also an extension function. (Which probably is the better idea.)